<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm_XbeeNetworkMessenger</class>
 <widget class="QMainWindow" name="ihm_XbeeNetworkMessenger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_analyzer">
       <attribute name="title">
        <string>Analyzer</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="trace_active">
          <property name="text">
           <string>Active trace</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="clear_trace">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTextEdit" name="analyser"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_generator">
       <attribute name="title">
        <string>Generator</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QSpinBox" name="ID">
          <property name="toolTip">
           <string>Message ID</string>
          </property>
          <property name="prefix">
           <string>0x</string>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="displayIntegerBase">
           <number>16</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="dlc">
          <property name="toolTip">
           <string>Message lenght</string>
          </property>
          <property name="maximum">
           <number>99</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="datas">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Données utiles à envoyer.&lt;/p&gt;&lt;p&gt;Format : Decimal / hexa ou mixte, séparation par des espaces.&lt;/p&gt;&lt;p&gt;Ex: 100 0xab 12 245 0xef 0x9F 45&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="send">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Message</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QComboBox" name="message_list"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Destination</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QComboBox" name="node_list"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_diag">
       <attribute name="title">
        <string>Diag</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QGridLayout" name="layout_led_presence"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
